### DISCUSSION TITLE
如何导入mobaxterm的书签

### DISCUSSION THREAD

**Discussion by zhangyuming** [electerm/electerm#3590] [General]
目前我使用的是mobaxterm  想切换到electerm， 但是我的session太多了，手动一个个迁移麻烦， electerm能否导入mobaxterm的session呢？ 或者有无其他工具导入呢

**Comment by zxdong262** [comment]
目前没有工具，要是mobxterm可以导出的话，可以跟electerm导出的数据对比看看，应该是很容易用ai转换一下的

**Comment by sjcjack** [comment]
先导出一个空白的js 然后把这个js文件扔给deepseek 再把你从其他ssh工具导出的ssh配置也扔给deepseek 让他给你改写
